testnet/stacks-node/src/main.rs

```diff
@@ -432,10 +434,10 @@ fn main() {
 
     let num_round: u64 = 0; // Infinite number of rounds
 
-    if conf.burnchain.mode == "helium" || conf.burnchain.mode == "mocknet" {
-        let mut run_loop = helium::RunLoop::new(conf);
+    if conf.burnchain.mode == "regtest" || conf.burnchain.mode == "mocknet" {
```

Comment:
Thanks @ldiego08 for opening this PR, I understand the reason behind combining `regtest` and `mocknet` as a `local testnet` based on this type of check. I'm curious what do you think for the [bitcoin_regtest_controller.rs](https://github.com/ldiego08/stacks-core/blob/chore/rename-helium-to-local-testnet/testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs) file as it seems to be specifically related to transitioning from `helium` to `regtest`, with no explicit mention of `mocknet`.

If the intention in `bitcoin_regtest_controller.rs` is to refer to both `mocknet` and `regtest`, could you update the file name accordingly, along with any other references that would be affected by this change? Or otherwise rename in that file specifically to `regtest` instead of `local testnet`?

Reply:
Specifically to this code, `BitcoinRegtestController` is used [only for regtest](https://github.com/stacks-network/stacks-core/blob/424b13b1b19225571c9b5801db46c422c6aaed48/testnet/stacks-node/src/run_loop/local.rs#L47). There's a separate controller for mocknet, [MocknetController](https://github.com/stacks-network/stacks-core/blob/3fcd23f837d740125b5170583eed00c6a85b55db/testnet/stacks-node/src/burnchains/mocknet_controller.rs#L29). However, looking at the main runloop (nakamoto), it does use `BitcoinRegtestController` for all Bitcoin networks (including mainnet) except mocknet. Probably, at some point, devs figured the regtest controller worked for all other networks, so they reused it. It was never renamed to `BitcoinController` or something less specific though. 

It makes more sense to do the following renaming:

- `BitcoinRegtestController` → `BitcoinBurnchainController` — `BurnchainController` is the "parent" struct or generic version, so adding "burnchain" to the name makes sense. It can be kept as `BitcoinController` too.

- `MocknetController` → `BitcoinMocknetBurnchainController` — same as above about adding "burnchain". This controller seems specific to Bitcoin, so I would add that to the name too.

These changes trascend the scope of this PR, so I could work on it on a separate one. There are other naming in these controllers that could be normalized further to keep consistency.

Interested in knowing your thoughts @ASuciuX (and @wileyj)!

Reply:
Thanks for sharing! I'll close this conversation point as it is out of this PR scope.